cmd/agentbasedinstaller/register.go

```diff
@@ -293,6 +293,10 @@ func getReleaseVersionAndCpuArch(log *log.Logger, releaseImage string, releaseMi
 	if len(cpuArchs) == 0 {
 		return "", "", errors.New("could not get release architecture")
 	}
+	if len(cpuArchs) > 1 {
+		log.Info("multi arch release payload detected")
+		return version, "multi", nil
```

Comment:
Can we use https://github.com/openshift/assisted-service/blob/d3d92b22a27cc76041a729a6d2d6f49553828a32/internal/common/common.go#L54